scripts/mod_loader/__scripts.lua

```diff
@@ -1,7 +1,7 @@
 local scripts = {
 	"hard-alter",
-	"Cutils",
 	
+	"cutils",
```

Comment:
I think Git is not a particular fan of renaming that only changes capitalization. I have seen people do it before by deleting or a more significant rename, then renaming it back in 2 commits.

Reply:
Hm.. So I see. I tried to change the file to all lower case to match the other files there, but it doesn't look like git allowed that.
Anyways; having trimmed the loading of `cutils.dll` down to a single line of code, maybe it could simply go into another file instead.